cnf-certification-test/identifiers/identifiers.go

```diff
@@ -272,6 +272,15 @@ var (
 		Url:     formTestURL(common.PlatformAlterationTestKey, "isredhat-release"),
 		Version: versionOne,
 	}
+	TestUndeclaredContainerPortsUsage = claim.Identifier{
+		Url:     formTestURL(common.NetworkingTestKey, "undeclared-container-ports-usage"),
+		Version: versionOne,
+	}
+	// TestClusterCsiInfoIdentifier list Cluster CSIdriver Identifier retrieves Third Party CSI driver info.
+	TestClusterCsiInfoIdentifier = claim.Identifier{
+		Url:     formTestURL(common.DiagnosticTestKey, "cluster-csi-info"),
+    Version: versionOne,
+	}
```

Comment:
This two identifiers are not needed. The "undeclared-container-ports-usage" was already defined on line 289, and we don't have any test case for the "cluster-csi-info" one. Please remove them.

Reply:
I had a conflict so I thought should leave it at that.
Anyway, I fixed it.